nav2_collision_monitor/test/collision_monitor_node_test.cpp

```diff
@@ -579,6 +610,8 @@ TEST_F(Tester, testProcessStopSlowdown)
   ASSERT_NEAR(cmd_vel_out_->linear.x, 0.5 * SLOWDOWN_RATIO, EPSILON);
   ASSERT_NEAR(cmd_vel_out_->linear.y, 0.2 * SLOWDOWN_RATIO, EPSILON);
   ASSERT_NEAR(cmd_vel_out_->angular.z, 0.1 * SLOWDOWN_RATIO, EPSILON);
+  ASSERT_EQ(action_state_->action_type, SLOWDOWN);
```

Comment:
For correct test work, we need ensure that `action_state_` was received. For this it is need to:
1. Initialize action_state_ with `nullptr`, as made it for `cmd_vel_out_` in `Tester::publishCmdVel()` routine
2. Add new `Tester::waitActionState()` method like `Tester::waitCmdVel()`
3. Use this method before checking of `action_state_` propierties in each testcase